### PR TITLE
Resolve easy issues

### LINKED ISSUE #2: Definition of KT
The Introduction says, "Any malicious entries added to such a log will generally be visible to all users, in which case a user can detect that they're being impersonated by viewing the public keys attached to their account." In the following paragraph, privacy is mentioned: "KT can better preserve user privacy by only showing entries of the log to participants that genuinely need to see them." It is unclear what it means to say that all entries are generally visible to all users, while achieving privacy through access control. Maybe the sentence can be rephrased to: "Any malicious entries added to such a log will be visible to the owners and recipients of the public-keys, in which case a user can detect that they're being impersonated by viewing the public keys attached to their account".

### PR DESCRIPTION
**Pull request by Bren2010**
Closes #2, #3, #4
/cc @kevinlewi @eshaghoshms

### CODE REVIEW

File: draft-ietf-keytrans-architecture.md [hunk 1]
```diff
@@ -65,8 +65,9 @@ use them to impersonate or eavesdrop on conversations with that user.
 
 Key Transparency (KT) solves this problem by requiring the service operator to
 store user public keys in a cryptographically-protected append-only log. Any
-malicious entries added to such a log will generally be visible to all
-users, in which case a user can detect that they're being impersonated
+malicious entries added to such a log will generally be equally visible to both
+the key's owner and the owner's contacts,
+in which case a user can detect that they're being impersonated
```
Comment (OR13) [inline draft-ietf-keytrans-architecture.md:70]:
```suggestion
in which case a user can detect that they are being impersonated
```

File: draft-ietf-keytrans-architecture.md [hunk 2]
```diff
@@ -165,8 +166,7 @@ operations that can be executed by a user are as follows:
    returns the corresponding value and a proof of inclusion.
 2. **Update:** Adds a new key-value pair to the log, for which the server
    returns a proof of inclusion. Note that this means that new values are added
-   to the log immediately and are not queued for later insertion with a batch of
-   other values.
+   to the log immediately, and no provisional inclusion proof (like an SCT in {{RFC6962}}) is provided.
```
Comment (OR13) [inline draft-ietf-keytrans-architecture.md:169]:
```suggestion
   to the log immediately, and no provisional inclusion proof, such as an SCT as defined in {{Section 4.4 of RFC9162}} is provided.
```

File: draft-ietf-keytrans-architecture.md [hunk 3]
```diff
@@ -256,10 +299,53 @@ keys of regular contacts).
...
+Applications that rely on a Transparency Log deployed in Contact Monitoring mode
+MUST regularly engage in out-of-band communication
+({{out-of-band-communication}}) to ensure that they detect forks in a timely
+manner.
+
+Applications that rely on a Transparency Log deployed in either of the
+third-party modes SHOULD allow users to enable a "Contact Monitoring Mode". This
+mode, which affects only the individual client's behavior, would cause the
+client to behave as if its Transparency Log was deployed in Contact Monitoring
+mode. As such, it would start retaining state about previously looked-up keys
```
Comment (OR13) [inline draft-ietf-keytrans-architecture.md:311]: perhaps "resolved" instead of "looked-up" ?

**Review by OR13 — APPROVED** [review]

**Comment by kevinlewi** [comment]
I think this closes #2, #3, and #4, but not #6 right? Is #6 addressed in this PR (did I miss it)?

**Comment by Bren2010** [comment]
Yes, #6 is not addressed. I made a comment in that ticket addressing why it was removed

**Pull request merged**